test-tap/assert.js

```diff
@@ -1608,22 +1674,22 @@ test('.notThrowsAsync()', gather(t => {
 	});
 }));
 
-test('.notThrowsAsync() returns undefined for a fulfilled promise', t => {
+test('.notThrowsAsync() returns true for a fulfilled promise', t => {
 	return assertions.notThrowsAsync(Promise.resolve(Symbol(''))).then(actual => {
-		t.is(actual, undefined);
+		t.is(actual, true);
 	});
 });
 
-test('.notThrowsAsync() returns undefined for a fulfilled promise returned by the function', t => {
+test('.notThrowsAsync() returns true for a fulfilled promise returned by the function', t => {
```

Comment:
This is technically a change in the method contract, so it may be that we don't make this change to `notThrowsAsync` until the next major version.

If we don't make the change to `notThrowsAsync`, though, that kind of makes me wonder whether we should make the change to `notThrows` either for symmetry.